Makefile

```diff
@@ -49,6 +49,8 @@ start-test: ## Tests a previous build Docker image to see if starts
 	@#	If container name doesn't exist, echo, and remove the container
 	@docker ps | grep -q "${NAME_TEST_CONTAINER}" || \
 		(echo "[test] Container exited and failed." && \
+		(docker rm -f "${NAME_TEST_CONTAINER}" &>/dev/null && \
+		false)
```

Comment:
&& false? so we always want to return a failing exist status?

Reply:
Here yes, because this checks whether the container stayed up and didn't crash.
So we remove it all the while trying to preserve a failing status code